mk/config.make

```diff
@@ -13,7 +13,7 @@
 
 DEBUG=    FALSE
 OPT=      TRUE
-PARALLEL= NONE
+PARALLEL= MPIOMP
```

Comment:
Will this always be ON by default now ? What happens if MPI is not available locally ? For example, with conda builds, we only use serial GNU compilers. Though on second thought, they only use the autotools route

Reply:
Will switch back to NONE